smac/facade/epils_facade.py

```diff
@@ -339,6 +338,7 @@ def _get_rng(rng):
 
         # initialize random number generator
         if rng is None:
+            self.logger.info('no rng given, falling back to non-deterministic behaviour')
             num_run = np.random.randint(1234567980)
```

Comment:
Any reason why we use 1234567890 here and MAXINT below?
Also, isn't this function exactly the same as in smac_facade? If yes, please create a issue so we can move this in another PR.

Reply:
I agree that it should be MAXINT

Reply:
Now using MAXINT